store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -416,6 +416,12 @@ func flagsToStatementContext(flags uint64) *stmtctx.StatementContext {
 	sc.IgnoreTruncate = (flags & model.FlagIgnoreTruncate) > 0
 	sc.TruncateAsWarning = (flags & model.FlagTruncateAsWarning) > 0
 	sc.PadCharToFullLength = (flags & model.FlagPadCharToFullLength) > 0
+	sc.InInsertStmt = (flags & model.FlagInInsertStmt) > 0
+	sc.InSelectStmt = (flags & model.FlagInSelectStmt) > 0
+	sc.OverflowAsWarning = (flags & model.FlagOverflowAsWarning) > 0
+	sc.IgnoreZeroInDate = (flags & model.FlagIgnoreZeroInDate) > 0
+	sc.DividedByZeroAsWarning = (flags & model.FlagDividedByZeroAsWarning) > 0
+	// TODO set FlagInUpdateOrDeleteStmt, FlagInUnionStmt,
```

Comment:
Why not set the flag in this PR.

Reply:
`sc` has 
```go
	sc.InUpdateStmt
	sc.InDeleteStmt
```
But if we has `FlagInUpdateOrDeleteStmt`, we don't know whether `sc.InUpdateStmt` or `sc.InDeleteStmt` should be true.

`sc` has no inUnion flag